purchase_order_product_recommendation/__manifest__.py

```diff
@@ -12,6 +12,7 @@
     "installable": True,
     "depends": [
         "purchase",
+        "stock",
```

Comment:
Why this extra dependency?

Reply:
This module in the prevous version only depended on purchase and purchase depended on stock_account (https://github.com/odoo/odoo/blob/40db7a8cf25ef0b5dd5b9d89e3b92c19372b09f6/addons/purchase/__manifest__.py#L12), but in v12 purchase only depends on account, so we have to force the stock module dependency here

Reply:
You're right. It's done, thank you